js/game.js

```diff
@@ -0,0 +1,25 @@
+const app3=new Vue({
+    el : '#game',
+    data : {
+        game : []
+    },
+    created() {
+        fetch('https://ananya-agrawal.github.io/OpenCode-Collaborative/json/game.json')
```

Comment:
This is not good, so if you delete repo where will we get the json from 😄

Reply:
as mentioned earlier also it was for tym being as without this repo getting merged game.json doesnt exist so data cant be taken from it . it could only be changed after the creation of game.json in main repo
@gauthamzz